main.tf

```diff
@@ -24,6 +24,12 @@ resource "azurerm_resource_group" "rg" {
   tags = "${merge(var.common_tags,
     map("lastUpdated", "${timestamp()}")
     )}"
+  
+  # On creation, resource group is not ready without delay.
```

Comment:
I've never seen this before =/, is there an upstream bug with terraform?
Given we're going to be removing this resource I would leave this change out

Reply:
if you pass a resource group in from outside the module it will have the same affect